README.md

```diff
@@ -6,13 +6,28 @@ This is my educational project within IT KPI Python mentorship program by @webkn
 
 ## Installation
 
-Soon
+1. Clone this repository:
+```console
+git clone https://github.com/kpi-web-guild/django-girls-blog-Martolivna.git
+```
+or:
+```console
+git clone git@github.com:kpi-web-guild/django-girls-blog-Martolivna.git
+```
 
+2. Install requirements:
+```console
+pip install -r requirements/base.txt
```

Comment:
This will also require `-c` similar to earlier comments.

Reply:
Addressed https://github.com/kpi-web-guild/django-girls-blog-Martolivna/commit/bc7ccad9fd34b7ff4f82e76ced7740e0642f7741

Reply:
Marking this thread as resolved.